models/staging/stg_tokenprice_from_swaps.sql

```diff
@@ -0,0 +1,273 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key="key",
+        tags=['core', 'defi', 'tokenprice'],
+        cluster_by=['block_date', 'token_address']
+        )
+}}
+
+with 
+stables as (
+  select * from {{ ref('harmony_stable_tokens')}}
+),
+
+-- trim the swaps table, truncate to day, and normalize the decimals, remove nulls with inner join
+simpleswaps as (
+  select
+    date_trunc('day', s.block_timestamp) as block_date,
+    s.tx_hash,
+    s.pool_address,
+    s.token0_address,
+    s.token0_symbol,
+    s.token1_address,
+    s.token1_symbol,
+    (s.amount0in + s.amount0out) / pow(10, t0.decimals) as amt0,
+    (s.amount1in + s.amount1out) / pow(10, t1.decimals) as amt1
+  from {{ ref('swaps') }} as s
```

Comment:
I think we can add the incremental_last_x_days macro here. Be sure to test it tho...